doc/plot_conf.yml

```diff
@@ -0,0 +1,117 @@
+# 
+# Documentation's example plot settings
+#
+# A "default" key allows setting default values for each entry. All the
+# settings defined there can be overriden for specific methods.
+#
+# Each entry needs the following keys:
+#   trace: a trace *object* (use !call or !var tags provided by lisa.utils.Serializable)
+#   kwargs [optional]: keyword arguments to pass to the plot function. This may
+#       be mandatory for some plot functions that need a task or cpu name for
+#       example. Bear in mind that YAML is "typed", so 0 is not the same as "0"
+#   hide [optional]: Do not produce an example plot, in case no trace with
+#   necessary events is available for example.
+
+
+doc-plot-conf:
+    plots:
+        # Default values for the other entries
+        default:
+            trace: &trace0 !call:lisa.trace.Trace
+                trace_path: !env:interpolate $LISA_HOME/doc/traces/trace.dat
```

Comment:
That trace is pretty big (~7MB). The trace we use for Travis (`tests/assets/trace.txt`) is ~.5MB. It would be nice to consolidate these into a single trace, and if possible a not-too-big one. Although in the end both the `tests` and `doc` directories are outside of the `lisa` module directory, so it probably doesn't matter *too* much.

Reply:
I already cropped the trace to only contain the interesting (rt-app) section. The size comes from the fact lots of events were collected, so we can plot lots of things. As a compensation, I removed another unused trace that was ~90MB without any interesting events beyond sched_switch ( #981 )

tests/assets/trace.txt only contains sched_switches and wakeups, and 3 instances of sched_overutilized, so it cannot be used for plotting anything interesting.

Reply:
I also forgot that some tests are hardcoded to work with the test trace (e.g. overutilized duration), so it's probably best to keep test and doc traces separate anyway.